common/src/main/java/me/lucko/luckperms/common/locale/Message.java

```diff
@@ -2781,7 +2781,7 @@ static TextComponent prefixed(ComponentLike component) {
                     .append(OPEN_BRACKET)
                     .append(translatable("luckperms.command.user.info.uuid-type-key"))
                     .append(text(": "))
-                    .append(mojang ? translatable("luckperms.command.user.info.uuid-type.mojang", DARK_GREEN) : translatable("luckperms.command.user.info.uuid-type.not-mojang", DARK_GRAY))
+                    .append(floodgate ? text("Floodgate", GREEN) : mojang ? translatable("luckperms.command.user.info.uuid-type.mojang", DARK_GREEN) : translatable("luckperms.command.user.info.uuid-type.not-mojang", DARK_GRAY))
```

Comment:
Small nitpick, why is this `GREEN` rather than `DARK_GREEN` as the other options are?

Reply:
Just to differentiate it from the other options.